app/_hub/kong-inc/aws-lambda/_index.md

```diff
@@ -84,7 +101,12 @@ params:
       default: null
       value_in_examples: <LAMBDA_FUNCTION_NAME>
       datatype: string
-      description: The AWS Lambda function name to invoke.
+      description: |
+        The AWS Lambda function name to invoke. The name formats
+        of the function name can be name-only(`my-function`),
+        full ARN(arn:aws:lambda:us-west-2:123456789012:function:my-function) and
+        partial ARN(123456789012:function:my-function). You can also append a version
+        number or alias to any of the formats.
```

Comment:
```suggestion
      description: |
        The AWS Lambda function name to invoke. This may contain
        the function name only (`my-function`), the full ARN 
        (arn:aws:lambda:us-west-2:123456789012:function:my-function) or a
        partial ARN (123456789012:function:my-function). You can also append a version
        number or alias to any of the formats.
```